docs/flocker-features/pure-storage-configuration.rst

```diff
@@ -0,0 +1,13 @@
+.. _pure-storage-backend:
+
+=================================
+Pure Storage Block Device Backend
+=================================
+
+.. begin-body
+
+Pure Storage provides a Flocker backend plugin.
+
+For more information, including installation, testing and usage instructions, visit the `Pure Storage Flocker Driver on GitHub <https://github.com/sdodsley/purestorage-flocker-driver>`_
```

Comment:
Hey, if you want to update the URL, the official repo is at https://github.com/PureStorage-OpenConnect/purestorage-flocker-driver

Its private right now, but as soon as I've got the go-ahead I'll switch it to a public repo.